test/link/test.sh

```diff
@@ -106,6 +106,13 @@ done
 
 # These tests do their own thing
 
+i="scramble-romx.asm"
+startTest
+"$RGBASM" -o "$otemp" scramble-romx/a.asm
+rgblinkQuiet -o "$gbtemp" -S romx=3 "$otemp" >"$outtemp" 2>&1
+tryDiff scramble-romx/out.err "$outtemp"
+(( rc = rc || $? ))
```

Comment:
This should also check that the output binary is 4 banks big.

Reply:
It is not! :P

Reply:
It is now. :)